examples/module-code-splitting/webpack.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
I cannot let my webpack 5.46 emit `import(...)` with this config. Is this feature released?

Reply:
Oh sorry I forgot to export config file